azure/table_azure_api_management_backend.go

```diff
@@ -0,0 +1,225 @@
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/apimanagement/mgmt/2020-12-01/apimanagement"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+//// TABLE DEFINITION ////
+
+func tableAzureAPIManagementBackend(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_api_management_backend",
+		Description: "Azure API Management Backend",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"backend_id", "resource_group", "service_name"}),
+			Hydrate:    getAPIManagementBackend,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"ResourceNotFound", "ResourceGroupNotFound"}),
+			},
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listAPIManagements,
+			Hydrate:       listAPIManagementBackends,
+		},
+		Columns: azureColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Type:        proto.ColumnType_STRING,
+				Description: "A friendly name that identifies an API management backend.",
+			},
+			{
+				Name:        "id",
+				Description: "Contains ID to identify an API management backend uniquely.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromGo(),
+			},
+			{
+				Name:        "url",
+				Description: "Runtime Url of the Backend.",
```

Comment:
Could you please ensure consistent usage of the term `API management backend` across all column descriptions?